### ISSUE TITLE
Apple Silicon: Unsupported platform/architecture: darwin-arm64

### ISSUE THREAD

**Issue by TheRav3n** [caffco/get-video-duration#82]
Hi! Do you have plans to support Apple Silicon?

**Comment by Sumolari** [comment]
darwin-arm64 support depends on https://github.com/SavageCore/node-ffprobe-installer/issues/141. 

`get-video-duration` will support it as soon as `@ffprobe-installer/ffprobe` does.

**Comment by Beekiper** [comment]
Changed dependency with a fork of our own, since `node-ffprobe-installer` since it seems to be no longer maintained.

Until https://github.com/caffco/get-video-duration/pull/83 gets merged, you can use the following fork in package.json:

```json
"get-video-duration": "git+https://github.com/Honeyside/get-video-duration.git",
```

**Comment by Beekiper** [comment]
@Sumolari Any chance to merge that PR?

**Comment by Beekiper** [comment]
Could not wait anymore, as this package needs to be published on npm to work correctly. I fixed my PR code with some more commits and published a 3.0.3 version with Apple Silicon (darwin-arm64) support here: https://www.npmjs.com/package/get-video-duration-apple-silicon

Will delete it after the main repo supports Apple Silicon.

**Comment by Sumolari** [comment]
@Beekiper Sorry for the late response, I've been busy lately

Thanks for the PR. I've reviewed it and left a couple of comments, as soon as they are resolved I'm happy to merge it and release a new version of this package

Again, thanks for the help!

**Comment by Sumolari** [comment]
Apple Silicon support is added in v3.1.0.

Thanks @Beekiper for your contribution!

**Comment by Beekiper** [comment]
Thank you @Sumolari for the fast response1

I archived my fork on Github and deleted it from npm.

To everyone reading: this issue has been fixed in 3.1.0 and tested on MacBook Air M1. If you are experiencing something similar, something else happened!
